CITATION.cff

```diff
@@ -19,7 +19,11 @@ authors:
   family-names: Goudenhoofdt
   affiliation: Royal Meteorological Institute of Belgium
   orcid: https://orcid.org/0000-0002-6376-1515
-version: 0.3.0
-date-released: 2023-07-11
+- given-names: Hamid Ali
+  family-names: Syed
+  affiliation: Purdue University
+  orcid: https://orcid.org/0000-0002-7188-2544
+version: 0.3.1
+date-released: 2023-09-26
```

Comment:
I'm not confident enough that we can make a release today, but let's set this for 2023-09-28.